### PR TITLE
Fixed broken STTR files

### PR DESCRIPTION
**Pull request by robinpiets**
Hotfix: STTR files broke because Flo Legal updated their API without informing us 🌊

### CODE REVIEW

File: apps/client/package.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
     "fast-xml-parser": "^3.17.4",
     "import-sort-style-module": "^6.0.0",
     "jest-styled-components": "^7.0.2",
-    "lint-staged": "^10.2.9",
+    "lint-staged": "^10.3.0",
```
Comment (afjlambert) [inline apps/client/package.json]: Not part of this PR.

**Review by afjlambert — CHANGES_REQUESTED** [review]

File: apps/client/package.json (hunk 1 above)
Comment (robinpiets) [reply apps/client/package.json]: Otherwise I could not commit...

**Review by afjlambert — APPROVED** [review]
